docs/Project.toml

```diff
@@ -3,10 +3,12 @@ CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+PlutoStaticHTML = "359b1769-a58e-495b-9770-312e911026ad"
 TuringGLM = "0004c1f4-53c5-4d43-a221-a1dac6cf6b74"
 
 [compat]
 CSV = "0.9, 1"
 CategoricalArrays = "0.10"
 DataFrames = "1"
 Documenter = "0.27"
+PlutoStaticHTML = "3"
```

Comment:
Version 3? Isn't it `0.3.0`?

Reply:
No. I'm not ending up in https://0ver.org/ land. At 1.0, my package was stable and reliable. Since then, I've moved through a few super breaking changes so now we're at 3.0.0